framework/arcane-framework/src/main/scala/models/cdm/SimpleCdmModel.scala

```diff
@@ -0,0 +1,35 @@
+package com.sneaksanddata.arcane.framework
+package models.cdm
+
+import models.{ArcaneSchema, ArcaneSchemaField, ArcaneType, Field, MergeKeyField}
+import upickle.default.*
+
+import scala.language.implicitConversions
+
+case class SimpleCdmAttribute(name: String, dataType: String, maxLength: Int)
+  derives ReadWriter
+
+case class SimpleCdmEntity(
+                            @upickle.implicits.key("$type")
+                            entityType: String,
+                            name: String,
+                            description: String,
+                            attributes: Seq[SimpleCdmAttribute])
+  derives ReadWriter
+
+
+case class SimpleCdmModel(name: String, description: String, version: String, entities: Seq[SimpleCdmEntity])
+  derives ReadWriter
+
+object CdmConversions:
+  implicit def asField(entity: SimpleCdmEntity): ArcaneSchemaField = entity.entityType match
```

Comment:
Implement Implicit conversion as described here in scala-3 way: https://docs.scala-lang.org/tour/implicit-conversions.html